### PR TITLE
[zh-cn] update web/css/align-items translation

### PR DESCRIPTION
**Pull request by nygmaaa**
### Description
Update the zh-CN version of the Web/CSS/align-items documentation to align it with the en-US version and rectify certain statements.

### Motivation
I found that the translation is outdated compared to the en-US version, and some content, such as the explanation of values like `start`, `center`, `left`, `right`, and others, remains untranslated.

### Additional details
Some translations are referenced from https://drafts.csswg.org/css-align-3/

### Related issues and pull requests

Relates to #12632

### CODE REVIEW

**Comment by jasonren0403** [comment]
Let's wait for upstream changes(mdn/content#29686) before we get content review here.

**Comment by nygmaaa** [comment]
@jasonren0403 The upstream changes have been merged, and I have updated the translation to align with them, please help review again. Thanks.

File: files/zh-cn/web/css/align-items/index.md [hunk 1]
```diff
@@ -209,33 +210,35 @@ select {
...
 
 {{Compat}}
 
 ## 参见
 
-- [使用 CSS 弹性框](/zh-CN/docs/CSS/Using_CSS_flexible_boxes)
+- CSS Flexbox 指南：_[flex 布局的基本概念](/zh-CN/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox)_
+- CSS Flexbox 指南：_[对齐弹性容器中的弹性项目](/zh-CN/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container)_
+- CSS Grid 指南：_[网格布局中的盒模型对齐](/zh-CN/docs/Web/CSS/CSS_grid_layout/Box_alignment_in_grid_layout)_
+- [CSS Box Alignment](/zh-CN/docs/Web/CSS/CSS_box_alignment)
```
Comment (jasonren0403) [inline files/zh-cn/web/css/align-items/index.md:243]:
```suggestion
- [CSS 盒对齐](/zh-CN/docs/Web/CSS/CSS_box_alignment)
```

**Review by jasonren0403 — COMMENTED** [review]
一些建议

File: files/zh-cn/web/css/align-items/index.md [hunk 2]
```diff
@@ -46,40 +49,40 @@ align-items: unset;
...
+  - : 如果（多个）元素的组合大小小于容器的大小，其中自动调整大小的元素将等量增大，以填满容器，同时这些元素仍然保持其宽高比例的约束。
 - `safe`
-  - : Used alongside an alignment keyword. If the chosen keyword means that the item overflows the alignment container causing data loss, the item is instead aligned as if the alignment mode were `start`.
+  - : 与其它对齐值一起使用。如果所选对齐值导致元素溢出容器，则将元素按 `start` 方式对齐。
 - `unsafe`
-  - : Used alongside an alignment keyword. Regardless of the relative sizes of the item and alignment container and whether overflow which causes data loss might happen, the given alignment value is honored.
+  - : 与其它对齐值一起使用。不管元素和容器的相对尺寸以及是否会发生溢出，都会采用给定的对齐值。
 
-### 语法格式
+## 正式定义
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/web/css/align-items/index.md:81]: @nygmaaa 这里还有一项

以后可以在 Files changed 标签页中批量应用更改，这样提交记录会干净一些

image
image

[the remark this answers is not in the document]
Comment (nygmaaa) [reply files/zh-cn/web/css/align-items/index.md:81]: 感谢提醒，我之前还没用过这个功能。

**Review by jasonren0403 — APPROVED** [review]
感谢你的贡献，欢迎来到MDN 🙇‍

**Pull request merged**